content/workshops/example/workshop_content.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Workshop Content
+workshops: example
+workshop_weight: 30
+layout: lab
+---
+
+## SETTING EXPECTATIONS FOR YOUR AUDIENCE
+
+----
+
+- Clearly tell your audience what the lab is, what things they will be doing, and what kind of skills are required to be successful at your workshop.
+
+- Examples of expectation setting:
+	- “This is a hands-on workshop.”
+	- “You should be comfortable with the command line.”
+	- “You should be comfortable being a Java developer.”
+	- “You understand classes and how to create them.”
+	- “You can write, understand and create html/jsp/servlets.”
+
+
+- Teaching someone something about the why you are doing the things in you workshop and possibly some history on why this way is better compared to previous version. 
+
+- Do not replicate official Red Hat training. Workshops are to get the audience more interested in Red Hat products and services. They should leave learning something but not to the point they can go get a certification. 
+
+- The best workshops are the ones that can tell a story.  Addressing or overcome previous technological issues help to show the value of Red Hat products. 
+
+- Every lab should end with a writeup of what was accomplished. 
+
+## WORKSHOP DESIGN: Format
+
+----
```

Comment:
Don't put horizontal lines as it doesn't match the style of the rest of the labs in the example workshop.